extensions/io/org.eclipse.smarthome.io.mqttembeddedbroker.test/src/test/java/org/eclipse/smarthome/io/mqttembeddedbroker/internal/MqttEmbeddedBrokerServiceTest.java

```diff
@@ -84,7 +84,7 @@ public void checkServerTrusted(X509Certificate[] chain, String authType, SSLEngi
         }
     }
 
-    private EmbeddedBrokerService subject;
+    private EmbeddedBrokerServiceImpl subject;
```

Comment:
should the class be renamed to `EmbeddedBrokerServiceImplTest` then?

Reply:
We are also testing the interface here. I'd leave the name for now.